review/tests.py

```diff
@@ -51,3 +65,29 @@ def test_delete_related_dogowner(self, test_review):
         dogowner.delete()
         assert dogowner not in DogOwner.objects.all()
         assert test_review in Review.objects.all()
+
+    def test_get_review_by_daycare_id(self, test_daycare, test_dogowner):
+        reviews = Review.get_review_by_daycare_id(test_daycare.id)
+        assert len(reviews) == 0
+
+        review_list = []
+
+        review_list.append(Review.create(review='review 1', rating=1, daycare_id=test_daycare.id,
+                                         dogowner_id=test_dogowner.id))
+
+        reviews = Review.get_review_by_daycare_id(test_daycare.id)
+        assert len(reviews) == len(review_list)
+        assert reviews.first() == review_list[0]
+
+        review_list.append(
+            Review.create(review='review 2', rating=2, daycare_id=test_daycare.id, dogowner_id=test_dogowner.id))
+        review_list.append(
+            Review.create(review='review 3', rating=3, daycare_id=test_daycare.id, dogowner_id=test_dogowner.id))
+
+        reviews = Review.get_review_by_daycare_id(test_daycare.id)
+        assert len(reviews) == len(review_list)
+
+        i = 0
```

Comment:
this is not needed. you can use `[enumerate](https://realpython.com/python-enumerate/)`